objgraph.py

```diff
@@ -1208,43 +1216,43 @@ def _gradient(start_color, end_color, depth, max_depth):
     return h, s, v
 
 
-def _edge_label(source, target, shortnames=True):
+def _edge_labels(source, target, shortnames=True):
     if (_isinstance(target, dict)
             and target is getattr(source, '__dict__', None)):
-        return ' [label="__dict__",weight=10]'
+        yield ' [label="__dict__",weight=10]'
     if _isinstance(source, types.FrameType):
         if target is source.f_locals:
-            return ' [label="f_locals",weight=10]'
+            yield ' [label="f_locals",weight=10]'
         if target is source.f_globals:
-            return ' [label="f_globals",weight=10]'
+            yield ' [label="f_globals",weight=10]'
     if _isinstance(source, types.MethodType):
         try:
             if target is source.__self__:
-                return ' [label="__self__",weight=10]'
+                yield ' [label="__self__",weight=10]'
             if target is source.__func__:
-                return ' [label="__func__",weight=10]'
+                yield ' [label="__func__",weight=10]'
         except AttributeError:  # pragma: nocover
             # Python < 2.6 compatibility
             if target is source.im_self:
-                return ' [label="im_self",weight=10]'
+                yield ' [label="im_self",weight=10]'
             if target is source.im_func:
-                return ' [label="im_func",weight=10]'
+                yield [' [label="im_func",weight=10]'
```

Comment:
```suggestion
                yield ' [label="im_func",weight=10]'
```

otherwise this is a syntax error.

(Are you running the tests locally before pushing?)